src/OptionList.tsx

```diff
@@ -20,7 +21,7 @@ export type OptionListProps = Record<string, never>;
 export interface RefOptionListProps {
   onKeyDown: React.KeyboardEventHandler;
   onKeyUp: React.KeyboardEventHandler;
-  scrollTo?: (index: number) => void;
+  scrollTo?: (args: number | ScrollConfig) => void;
```

Comment:
Since [the virtual list here](https://github.com/react-component/virtual-list/blob/master/src/List.tsx#L35)used `(arg: number | ScrollConfig)` as its params type, so I think maybe we should allow here to accept type ScrollConfig